impeller/entity/inline_pass_context.cc

```diff
@@ -12,9 +14,9 @@
 namespace impeller {
 
 InlinePassContext::InlinePassContext(std::shared_ptr<Context> context,
-                                     RenderTarget render_target,
+                                     const RenderTarget& render_target,
                                      uint32_t pass_texture_reads)
-    : context_(context),
+    : context_(std::move(context)),
```

Comment:
This is is now a copy. The right fix here is to add the move but remove the const reference in the argument.